src/SFA.DAS.EAS.Web/Orchestrators/EmployerCommitmentsOrchestrator.cs

```diff
@@ -1047,20 +1059,13 @@ private Task<GetAccountTransferConnectionsResponse> GetTransferConnectionsNoAuth
                 };
 
             }, hashedAccountId, externalUserId);
-
         }
 
-
         public async Task<bool> AnyCohortsForCurrentStatus(string hashedAccountId, params RequestStatus[] requestStatusFromSession)
```

Comment:
As I've rewritten this (to stop multiple calls to the commitments api, to not unnecessarily create a list and to short-circuit processing if Any found), I added unit test coverage in CheckingForAnyCohortsForCurrentStatus.cs